src/generator.ts

```diff
@@ -46,6 +63,8 @@ export function generate(node: t.Node | t.PropTypeNode[], options: GenerateOptio
 		sortProptypes = true,
 		importedName = 'PropTypes',
 		includeJSDoc = true,
+		previousPropTypesSource = new Map<string, string>(),
+		reconcilePropTypes = (_prop: t.PropTypeNode, _previous: string, generated: string) => generated,
```

Comment:
Does the typechecking fail if you don't type these?

Reply:
They would have implicit any otherwise.